packages/client/src/services/users.ts

```diff
@@ -135,8 +141,9 @@ const validate = async (
 };
 
 const update = async (id: string, partialUser: Partial<UserState>) => {
-  const response = await axios.put(`/users/${id}`, partialUser);
-  return response;
+  // TODO: update firebase credentials somehow
```

Comment:
hmmmm this seems important @jpfranci thoughts?

Reply:
I will add this soon. @jpfranci are we sticking with only email/password for authentication? (and leaving username as a display name thing)

Reply:
Yeah we can

Reply:
hmm I think it'd be ok if u merge this and then open another pr for fixes

(I def suggested this bc I worked on top of ur changes https://github.com/jpfranci/Alpacow/pull/30)